code/game/machinery/computer/ai_core.dm

```diff
@@ -264,9 +264,9 @@ atom/proc/transfer_ai(var/interaction, var/mob/user, var/mob/living/silicon/ai/A
 	if(interaction == AI_TRANS_FROM_CARD)
 		AI.control_disabled = 0
 		AI.aiRadio.disabledAi = 0
-		AI.loc = loc//To replace the terminal.
+		AI.forceMove(loc)//To replace the terminal.
 		to_chat(AI, "You have been uploaded to a stationary terminal. Remote device connection restored.")
-		to_chat(user, "<span class='boldnotice'>Transfer successful</span>: [AI.name] ([rand(1000,9999)].exe) installed and executed successfully. Local copy has been removed.")
+		to_chat(user, "<span class='boldnotice'>Transfer successful</span>: [AI.name] ([rand(1000,9999)].exe) installed and executed successfully. Local copy has been removed.</span>")
```

Comment:
The `</span>` you added doesn't seem to be connected to anything. What I'm saying message is probably going to display as something like "**Transfer successful**: [AI.name] ([rand(1000,9999)].exe) installed and executed successfully. Local copy has been removed.</span>"